.github/workflows/release.yaml

```diff
@@ -1,25 +1,38 @@
-# This is a template `release.yaml` file for ops charms
-# This file is managed by bootstack-charms-spec and should not be modified
-# within individual charm repos. https://launchpad.net/bootstack-charms-spec
-
 name: Release to Edge
 
 on:
   push:
-    branches: [master, main]
-
-concurrency:
-  group: release
-  cancel-in-progress: true
+    branches: [ main ]
+  release:
+    types: [ published ]
 
 jobs:
   check:
     uses: ./.github/workflows/check.yaml
+    secrets: inherit
 
   release:
     needs: check
-    uses: canonical/bootstack-actions/.github/workflows/charm-release.yaml@v2
-    secrets: inherit
-    with:
-      channel: "latest/edge"
-      upload-image: false
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Initialize lxd  # This should dropped once it's implemented on charming-actions itself. https://github.com/canonical/charming-actions/issues/140
+        if: ${{ inputs.destructive-mode == false }}
+        run: |
+          sudo lxd init --auto
+          # The following is needed to fix network error when creating new base instance on lxd
```

Comment:
is there a bug for this?

Reply:
Yes, I reported the [#140](https://github.com/canonical/charming-actions/issues/140) in official actions.

Reply:
yeah and it's good that you added that comment; I meant to ask if there's a bug for the networking-related workarounds you're doing. #140 does not mention anything network-related.

Reply:
I'm not sure why the network was needed, we need to ask @sudeephb, since he introduced it in [#47](https://github.com/canonical/bootstack-actions/pull/47).

Reply:
It's the same steps in https://github.com/charmed-kubernetes/actions-operator/blob/f33f1c1cf6adc74426e49f4e293a41f924f20d0a/src/bootstrap/index.ts#L231

Reply:
The reference was taken from [here](https://github.com/charmed-kubernetes/actions-operator/blob/df660b2ba29176d8b638e51c40240b4bc9bd58c4/src/bootstrap/index.ts#L222). So, while this is needed for lxd instance to the external networks, it seems to be handled like it's intended behavior. 
I recall I also got this section of the action from somewhere else ( 🫣 ), let me try and find it.

Reply:
that code is very old, are you sure it's still necessary? Have you tried without it?

Reply:
thank you, a lot cleaner!